src/models/OtrStore.js

```diff
@@ -0,0 +1,67 @@
+var _ = require("lodash");
+var DSA = require('otr').DSA;
+var OTR = require('otr').OTR;
+var Chan = require("./chan");
+
+module.exports = OtrStore;
+
+/** Stores OTR state for a given user
+ *
+ * Handels private key, current sessions and fingerprints.
+ */
+function OtrStore(name, client) {
+	_.merge(this, {
+		name: name,
+		privKey: undefined,
```

Comment:
By default, non-existing properties of an object are already `undefined`:

```js
$ node
> var s = new String;
undefined
> s
[String: '']
> s.foobar
undefined
```

The fact that they do not throw `ReferenceError` exceptions as do variables can be [a big pain in some cases](https://github.com/moll/js-must#asserting-on-property-access) but in your case, it means you don't need this line.